Directory.Build.props

```diff
@@ -34,6 +34,7 @@ Some common use cases include:
         <!-- https://www.gresearch.co.uk/blog/article/improve-nuget-restores-with-static-graph-evaluation/ -->
         <RestoreUseStaticGraphEvaluation>true</RestoreUseStaticGraphEvaluation>
         <ServerGarbageCollection>true</ServerGarbageCollection>
+        <LangVersion>PREVIEW</LangVersion>
```

Comment:
Is it `PREVIEW` in all caps? Or does that not matter at all?

Reply:
It doesn't matter. Normally, tooling inserts lowercase.

Reply:
Yes, changed it to lowercase to be more consistent.